requirements-dev.txt

```diff
@@ -28,3 +28,6 @@ sphinx~=4.4
 sphinx_rtd_theme~=1.0
 tox>=3.14
 devtools~=0.6
+# temporarily until we have a pypi package
+cmake
```

Comment:
Just wondering: My quick understanding was that `scikit-build` was used so that `cmake` was not required. Is that not the case?